signalbot/message.py

```diff
@@ -123,6 +129,28 @@ async def parse(cls, signal: SignalAPI, raw_message_str: str) -> Message:  # noq
                     # Ignore them by raising a known error.
                     raise UnknownMessageFormatError
 
+                if "readMessages" in sync_message:
+                    return cls(
+                        source,
+                        source_number,
+                        source_uuid,
+                        timestamp,
+                        message_type=MessageType.READ_MESSAGE,
+                        text="",
+                        base64_attachments=[],
+                        attachments_local_filenames=[],
+                        view_once=view_once,
+                        link_previews=[],
+                        group=None,
+                        reaction=None,
+                        mentions=[],
+                        quote=None,
+                        read_messages=sync_message["readMessages"],
+                        target_sent_timestamp=target_sent_timestamp,
+                        remote_delete_timestamp=None,
+                        raw_message=raw_message_str,
+                    )
```

Comment:
How about simplifying the function call by using the defaults?

```suggestion
                    return cls(
                        source,
                        source_number,
                        source_uuid,
                        timestamp,
                        MessageType.READ_MESSAGE,
                        "",
                        read_messages=sync_message["readMessages"],
                        target_sent_timestamp=target_sent_timestamp,
                        raw_message=raw_message_str,
                    )
```